dali/core/mm/default_resource_test.cc

```diff
@@ -379,7 +379,9 @@ static void ReleaseUnusedTestImpl(ssize_t max_alloc_size = std::numeric_limits<s
 }
 
 TEST(MMDefaultResource, ReleaseUnusedBasic) {
-  ReleaseUnusedTestImpl(256 << 20);
+  size_t free, total;
+  CUDA_CALL(cudaMemGetInfo(&free, &total));
+  ReleaseUnusedTestImpl(align_down(free / 2, 64 << 20));
```

Comment:
This change was necessary because some systems (Orin) didn't even notice the 256M allocation.